Sources/FoundationEssentials/FileManager/FileManager+SymbolicLinks.swift

```diff
@@ -40,14 +42,35 @@ extension _FileManagerImpl {
         guard !destPath.isEmpty else {
             throw CocoaError.errorWithFilePath(.fileNoSuchFile, destURL)
         }
-        
-        try createSymbolicLink(atPath: path, withDestinationPath: destPath)
+
+#if os(Windows)
+        let bIsDirectory = try path.withNTPathRepresentation {
```

Comment:
Instead of calling `GetFileAttributesExW` here should we just do

```swift
var bIsDirectory = false
_ = self.fileExists(atPath: path, isDirectory: bIsDirectory)
```

So that we don't duplicate that logic?

Reply:
Yeah, that probably makes sense to share the implementation. We only care about the directory type flag.